docs/extend/plugins.md

```diff
@@ -34,7 +34,7 @@ Follow the instructions in the plugin's documentation.
 The sections below provide an inexhaustive overview of available plugins.
 
 <style>
-#content tr td:first-child { white-space: nowrap;}
+#DocumentationText  tr td:first-child { white-space: nowrap;}
```

Comment:
@thajeztah   I have changed, but the result is still not correct.  it looks like:

![image](https://cloud.githubusercontent.com/assets/17755856/16862756/14d4c1d0-4a7f-11e6-90da-16beefb489c7.png)